templates/vault/0/rancher-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: '2'
+catalog:
+  name: Vault
+  version: 1.0-test-rancher1
+  description: |
+    Installs a Vault container, internal (Rancher) load-balancer, and allows connection to a Consul cluster
+  uuid: vault-rac-0
+  questions:
+    - variable: VAULT_LOCAL_CONFIG
+      label: Provide a default config?
+      description: |
+        This is the embedded JSON format config file that vault will use at startup
+      type: multiline
+      default: |
+        {"backend":{"consul":{"address":"http://<listenAddress>:<listenPort>", "advertise_addr":"http://<advertiseAddress>","path":"<Path>"}}, "listener":{"tcp":{"address":"0.0.0.0:18200","tls_disable":1}}}

```

Comment:
Default value didn't work. What do you think to add a `templates/vault/0/README.md` to explain who to set it??

Reply:
I'm not sure how to fix this one.  It seems that when the template renders in Rancher, it always puts single-quotes around the value specified.  So, even though the default is set without them, once the page renders, you'll see the value as '{"backend":... ..._disable:1}}}' <- Notice the single quotes before and after the value.  If you manually strip those off, and set the proper values, it works just fine.  Again, I'm simply not sure how to tell rancher not to quote the default value.

Reply:
Done;  Added a README.md, and fixed an issue with single quotes being added around the default configuration.  It should work without modification, now.